### ISSUE TITLE
[BUG] Insufficient contrast ratio for Update button 

### ISSUE THREAD

**Issue by risanaka** [microsoft/accessibility-insights-windows#342]
Please check whether the bug has [already been filed](https://github.com/Microsoft/accessibility-insights-windows/issues).

**Describe the bug**

When user is prompted to update Accessibility Insights for Windows, when they hover over the Update button, it does not have sufficient contrast.
**To Reproduce**
1. Install Accessibility Insights for Windows version that's earlier than the minimum.
1. Launch Accessibility Insights for Windows.

**Expected behavior**

The update button in the update menu bar, on hover, should maintain minimum contrast per [WCAG 2.1 AA Success Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).

**Actual behavior**
What actually happened.

The update button in the update menu bar, on hover, loses minimum contrast.

**Screenshots or .GIF**

![image](https://user-images.githubusercontent.com/144273/56915587-ca531c00-6a6b-11e9-909a-db2162e2d28e.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10 1809] Windows 10 version 1809
 - Accessibility Insights for Windows Version: 1.1.837.2
 - Target Application: [e.g. Chrome, Wildlife Manager] n/a
 - Target Application Version: [e.g. 22] n/a

**Additional context**
Priority requested -

**Comment by jalkire** [comment]
Do we have any guidance as to what the colors should be on hover? @cheade

**Comment by cheade** [comment]
@jalkire Hi!

Here are the colors for the button states:
![Button state colors](https://user-images.githubusercontent.com/40274682/57100808-b1996f00-6cd4-11e9-8710-84b216ff31e3.png)

**Comment by risanaka** [comment]
@jalkire thanks, I'll review and close it once I see it in the Canary build next week :)
